core/domain/story_services.py

```
@@ -984,7 +984,9 @@ def delete_story_summary(story_id: str) -> None:
             be deleted.
     """
 
-    story_models.StorySummaryModel.get(story_id).delete()
+    story_summary_model = story_models.StorySummaryModel.get(story_id)
```

I think we should add strict=False param; otherwise, it will still fail noisily if the StorySummaryModel instance doesn’t exist.